.github/templates/model_regression_test_results.tmpl

```diff
@@ -0,0 +1,15 @@
+{{ range $dataset, $config := (datasource "data")}}
+Dataset: `{{$dataset}}`
+
+| Configuration | Intent accuracy | Entity F1 Score | Response accuracy |
```

Comment:
optional: break line after 80 / 88 chars (similar as we do it in Python) / same for other files

Reply:
In templates is not always possible, templates take everything inside of them very literally